### ISSUE TITLE
Remove use of average load and of strict dichotomy between underloaded and overloaded processors

### ISSUE THREAD

**Issue by ppebay** [DARMA-tasking/LB-analysis-framework#103]
@lifflander @PhilMiller  this should reflect our design discussion of 11/10/21; please comment as you see fit

One side effect will be to no longer have the ability to compare with the original "Grapevine" algorithm

**Comment by ppebay** [comment]
Preliminary results after removal of underloaded/overloaded subsetting:

```
Descriptive statistics of final rank loads:
	cardinality: 8  sum: 1.85661  imbalance: 0.0340546
	minimum: 0.224645  mean: 0.232076  maximum: 0.23998
	standard deviation: 0.0044011  variance: 1.93697e-05
	skewness: 0.288683  kurtosis excess: -0.434795
```

<img width="726" alt="Screen Shot 2021-12-08 at 3 12 06 PM" src="https://user-images.githubusercontent.com/17439260/145222884-ba5467a9-bd58-476d-a2c4-8645fb51796b.png">

**Comment by ppebay** [comment]
This one is long-resolved, in particular by PR #124
